ambari-web/app/views/wizard/step2_view.js

```diff
@@ -140,6 +131,32 @@ App.WizardStep2View = Em.View.extend({
      * @type {bool}
      */
     disabled: Em.computed.not('isEnabled')
-  })
+  }),
+
+  /**
+   * Filter for pre-registered hosts. Currently this is a text filter.
+   *
+   * Need to change it to column filter at some point
+   */
+  filterHostsTable: function () {
+    var self = this;
+
+    var filter = this.get('controller.filterText');
+    if (filter) {
+      filter = filter.toUpperCase();
+    }
+    this.get('controller.manuallyInstalledHosts').forEach(function (host) {
```

Comment:
We are trying to minimize directly changing DOM nodes, use class binding instead.

Reply:
@atkach But here I need to dynamically set display to none or block based on what is being typed in the filter box for each item in the list. I thought fetching by id is quick. How do I bind each item in the list to a class?